raphtory/src/python/graph/algorithm_result.rs

```diff
@@ -312,6 +312,12 @@ py_algorithm_result_new_ord_hash_eq!(
     Vec<(i64, String)>
 );
 
+py_algorithm_result!(AlgorithmResultVecF64, DynamicGraph, Vec<f64>, Vec<f64>);
+// NOTE(Wyatt): This is currently just used by FastRP and it doesn't really make sense to implement max, min, median, etc. methods on the output.
```

Comment:
Yes!

Reply:
I suppose computing the bounding box and centroid of an embedding could be occasionally useful. All it would take to get this working is to change the types everywhere and implement `AsOrd` for `Vec<T: FloatCore>`.